module/dice/d20-roll.mjs

```diff
@@ -251,7 +253,21 @@ export default class D20Roll extends Roll {
     // Customize the modifier
     if ( form.ability?.value ) {
       const abl = this.data.abilities[form.ability.value];
+      let prof = this.data.baseProf;
+
+      // Make proficiency adjustments based on remarkable athlete
+      if ( this.options.remarkableAthlete ) {
```

Comment:
I don't like that something as specific as remarkable athlete interjects itself into the fundamental rolling workflow for every `D20Roll` throughout the entire system. I know that for specific types of checks we handle remarkable athlete further upstream. Is there any other way to approach this? I am uncomfortable with this design.

Reply:
Hmm, we are already swapping out bonuses when the ability is changed.

I suppose we could add a hook here to allow skills to customize the roll.